channel/channel.go

```diff
@@ -36,8 +35,8 @@ type Channel struct {
 // New constructs a new Channel from the supplied state.
 func New(s state.State, isTwoPartyLedger bool, myIndex uint, myDestination types.Destination, theirDestination types.Destination) (Channel, error) {
 	c := Channel{}
-	if s.TurnNum.Cmp(big.NewInt(0)) != 0 {
-		return c, errors.New(`objective must be constructed with a turnNum 0 state`)
+	if s.TurnNum != 0 {
```

Comment:
should `0` be an enum `PreFundTurnNum`?

Reply:
Yeah I think that's a good suggestion. Will raise a new PR.

Reply:
#150